### PR TITLE
feat(RHTAPREL-806): update operator toolkit refrence

### PR DESCRIPTION
**Pull request by happybhati**
### What does this PR do?:
 This commit updates the `operator-toolkit` refrence
 as a result of migration to `konflux-ci` from `redhat-appstudio`
 More details parent EPIC:
 https://issues.redhat.com/browse/RHTAPREL-800

### Which issue(s)/story(ies) does this PR fixes:
  
https://issues.redhat.com/browse/RHTAPREL-806

### CODE REVIEW

**Comment by happybhati** [comment]
`Do Not Merge/Hold` preparation PR, Can be merged once repo is migrated to `konflux-ci`!

**Review by maysunfaisal — APPROVED** [review]
/lgtm

**Pull request merged**
